src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -1368,7 +1368,7 @@ MulticopterAttitudeControl::start()
 	_control_task = px4_task_spawn_cmd("mc_att_control",
 					   SCHED_DEFAULT,
 					   SCHED_PRIORITY_MAX - 5,
-					   1700,
+					   2200,
```

Comment:
I would assume this affects every task and every xxxxx_main run off of init. 

We need to avoid #ifdef rash!

To isolate the the ARM build from clang induced inefficiencies, how about:

1. add a clang wrapper for  "px4_task_spawn_cmd' that added some constant. 
Or 
2. Do a separate PR  against master that adds PX4_SET_STACK(x) to all px4_task_spawn_cmd calls. on master it can be just ```#define PX4_SET_STACK(x) (x)``` Then on the clang build define it as ```#define PX4_SET_STACK(x) (x)+BUILD_CLANG_STACK_ADDITIONAL```?  

Both options opens the door for 
```#define PX4_SET_STACK(x) (x)+BUILD_xxx_STACK_ADDITIONAL+ARCH__xxx_STACK_ADDITIONAL``` 

But 2 can be added to the xxx_main invocations as well if the ARCH__xxx_STACK_ADDITIONAL and  BUILD_xxx_STACK_ADDITIONAL can come in from CMake